coursedata/level-defaults/en.yaml

```diff
@@ -499,7 +499,7 @@
 13:
```

Comment:
Interesting that this snippet dit not give an error! (this is a note to myself :)